Firestore/core/src/local/local_serializer.cc

```diff
@@ -136,7 +136,7 @@ google_firestore_v1_Document LocalSerializer::EncodeDocument(
   result.name = rpc_serializer_.EncodeKey(doc.key());
 
   // Encode Document.fields (unless it's empty)
-  const google_firestore_v1_MapValue& fields_map = doc.data().Get().map_value;
+  const google_firestore_v1_MapValue& fields_map = doc.value().map_value;
```

Comment:
Please make `fields_map` a value instead of a reference.

Reply:
Done